datalad_fuse/tests/conftest.py

```diff
@@ -16,12 +16,12 @@
 
 DATA_DIR = Path(__file__).with_name("data")
 
-lgr = logging.getLogger("datalad_fuse.tests")
+lgr = logging.getLogger("datalad.fuse.tests")
```

Comment:
yeah, we should do that. Filed also https://github.com/datalad/datalad-extension-template/issues/27 to "harmonize" in the future